Hackney.Shared.Asset/Infrastructure/AssetDb.cs

```diff
@@ -24,7 +24,7 @@ public class AssetDb
         public string RootAsset { get; set; }
 
         [DynamoDBProperty]
-        public bool IsActive { get; set; }
+        public bool? IsActive { get; set; }
```

Comment:
Is this parameter used anywhere? If its null, would you treat it as true or false? 

Perhaps instead of making it nullable, but defaulting it to true would make more sense? 🤔

Reply:
No idea, but if in use anywhere, I figured any app that has been using it, has been doing so assuming that the value can be either true or false, and not null.